website/blog/2023-10-29-databend-weekly-117.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "This Week in Databend #117"
+date: 2023-10-29
+slug: 2023-10-29-databend-weekly
+cover_url: 'weekly/weekly-117.jpg'
+image: 'weekly/weekly-117.jpg'
+tags: [weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: akoshchiy
+  - name: andylokandy
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTang
+  - name: dantengsky
+  - name: Dousir9
+  - name: drmingdrmer
+  - name: leiysky
+  - name: PsiACE
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zenus
+  - name: zhang2014
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's New
+
+Stay informed about the latest features of Databend.
+
+### Feature Preview: Read-Only ATTACH TABLE
```

Comment:
The feature is out now, as well as the document updates:) 
So I think we can get rid of "Feature Preview" from the title.